include/envoy/network/connection_handler.h

```diff
@@ -120,6 +120,9 @@ class ActiveUdpListenerFactory {
   virtual ConnectionHandler::ActiveListenerPtr
   createActiveUdpListener(ConnectionHandler& parent, Event::Dispatcher& disptacher,
                           spdlog::logger& logger, Network::ListenerConfig& config) const PURE;
+
+  // @return true if the protocol above UDP doesn't form a connection.
```

Comment:
super nitty but /** .. */ style comments
Also maybe 
"if the protocol above UDP doesn't form a connection." -> "If the UDP passing through listener doesn't form stateful connections" ?

might even be nicer to invert and have isStatefulConnection() but maybe that's just  me.

Reply:
done